test/model/src/core/kv_database.cpp

```diff
@@ -173,4 +173,130 @@ kv_database::txn_snapshot_nolock(txn_id_t do_not_exclude)
       _last_transaction_id, std::move(active_txn_ids));
 }
 
+/*
+ * kv_database::clear --
+ *     Clear the contents of the database.
+ */
+void
+kv_database::clear()
```

Comment:
I think I'm missing something - is this called from anywhere?

Reply:
That's right—it's not being used from anywhere, just the _nolock version. I can remove it.

Reply:
Yes please - it's not difficult to write if we ever need it.